### DISCUSSION TITLE
Gluetun / mullvad DDNS

### DISCUSSION THREAD

**Discussion by capsel22** [qdm12/gluetun#1353] [Q&A] [answered]
hi,

I'm new to VPNs, dockers and other things.
I was hoping someone could assist me or give me an idea how to use gluetun with mullvad port forwarding.

Issue I have is that the exit IP changes and I need to check the docker logs, what the new exit IP is to access the forwarded port.
Is there anyway to configure gluetun to update any free DDNS services like duckdns or smth when the exit ip changes?

Thanks,

**Comment by bnhf** [comment, marked as answer]
@capsel22 

Another project from the developer of Gluetun should be able to handle the updating of a DDNS service for you:

https://github.com/qdm12/ddns-updater

**Reply by capsel22** [reply]
Thanks will check it out.
